fake_data_generation/README.md

```diff
@@ -0,0 +1,58 @@
+# Generating fake data
+
+## Overview and Purpose
+This directory contains a file called `generate_fake_data.py`. The purpose of `generate_fake_data.py` is to create a `.csv` file with fake data with the following intended applications:
+- As an example of how data needs to be formatted to be passed into the model.
+- To ensure the files are being generated correctly to test GUI setup.
+- To test the setup and running of the repo.
+
+The following should be noted with regards to artefacts produced by using fake data:
+- *DO NOT* use the model artefacts (`.pickle` files) generated from the fake data to make predictions which will be used in any real world application.
+- *DO NOT* use the fake data generated to inform any insights to be applied to a real world setting.
+- *DO NOT* use the fake data to test the performance of the model.
+
+The data is generated completely randomly, with each field having random values generated independently of other fields. This generator was created having never been exposed to the real data.
+
+The categories for the fields in the fake data can be found in [fake_data_categories.json](../config/fake_data_categories.json). The categories specified in this file are randomly chosen to fill the corresponding fields in the fake data. This is done in `generate_fake_data.py`.
+
+Note a number of the fields will show only test categories and may not be reflective of the types of categories or number of categories types in each field. However the formatting of the fields is reflective as an example of what is required to run the model.
+
+Data fields that do not have their categories specified in [fake_data_categories.json](../config/fake_data_categories.json) or are not categorical variables have the fake data required generated in `generate_fake_data.py` line by line to show how the data is being generated for each field. In `generate_fake_data.py` the fields are split into the field data type (e.g. str, int) required for the models to train.
+
+
+## How to run
+Before running ensure your environment is set up as described in: [Training](../README.md#getting-started)   
```

Comment:
We don't have a Training section but do have a getting-started section - this will need an <a name="getting-started"></a> tag adding to it